vortex-array/src/view.rs

```diff
@@ -17,20 +19,25 @@ use crate::{Array, IntoArray, ToArray};
 #[derive(Clone)]
 pub struct ArrayView<'v> {
     encoding: EncodingRef,
-    dtype: &'v DType,
-    array: fb::Array<'v>,
-    buffers: &'v [Buffer],
-    ctx: &'v ViewContext,
+    dtype: DType,
+    flatbuffer: Buffer,
+    flatbuffer_loc: usize,
```

Comment:
Flatbuffer code doesn't generate traits for accessing the table/loc information. So we can't create some nice abstract wrapper for this stuff.